weatherman.py

```diff
@@ -0,0 +1,246 @@
+__weather__ = {}  # for storing weather records
+__months__ = {1: "Jan", 2: "Feb", 3: "Mar", 4: "Apr", 5: "May", 6: "Jun", 7: "Jul", 8: "Aug", 9: "Sep", 10: "Oct",
```

Comment:
Why we are hardcoding months and their sequence ourselves? Did we explored that Python have something relevant to this or not?
This is also against [DRY Principle](https://en.wikipedia.org/wiki/Don%27t_repeat_yourself)